src/core/utils/helpers/general.ts

```diff
@@ -545,4 +548,22 @@ if (import.meta.vitest) {
       );
     });
   });
+
+  describe("getParams", () => {
```

Comment:
is there a reason why we don't test in a separate file? Aren't the file sizes just gonna blow up this way?